content/blog/2017-11-10-rprofile-mara-averick.md

```diff
@@ -0,0 +1,127 @@
+
+KO: How often do you use R now?
+
+MA: Oh, I use R like every day. I use it… I don’t use Word any more. [Laughter] Yeah so one of the things about basketball is that there are times of the year where there are games every day. So that’s been my morning workflow for a while - scraping basketball data.
+
+KO: So you get up every morning and scrape what’s new in Basketball?
+
+MA: Yeah! So I end up in RStudio bright and early (often late, as well).
+
+KO: So is that literally what the first half hour of your day looks like?
+
+MA: No, so incidentally that’s kind of how this Twitter thing got started. My dog has long preceded me on Twitter and the internet at large, he’s kind of an internet famous dog [@batpigandme](https://twitter.com/batpigandme). There’s an application called [Buffer](https://buffer.com/) which allows you to schedule tweets and facebook page posts, which was most of Batpig’s traffic - [facebook page](https://www.facebook.com/batpigandme) visits from Japan. And so I had this morning routine (started in the winter when I had one of those light things you sit in front of for a certain number of minutes) where I would wake up and schedule batpig posts while I’m sitting there and read emails. And that ended up being a nice morning workflow thing.
+
+I went to a [Do Good Data](http://www.dogooddata.com/) conference, which is a [Data Analysts for Social Good](http://www.dataanalystsforsocialgood.com/) ([@DA4SG](https://twitter.com/DA4SG)) event, just over two years ago, and everyone there was giving out their twitter handles, and I was like, oh - maybe people who aren’t dogs also use Twitter? [Laughter] So that was how I ended up creating my own account [@dataandme](https://twitter.com/dataandme) independent from Batpig.
+
+KO: What happened after you went to this conference? Was it awesome, did it inspire you?
+
+MA: Yeah so, I was the stats person at the company I was working at. And I didn’t realize there was all this really awesome work being done with really rigorous evaluation that wasn’t necessarily federal grant proposal stuff.  So I was really inspired by that and started learning more about what other people were doing, some of it in R, some of it not. I kept in touch with some of the people from that conference. And then NBA Twitter is also a thing it turns out, and NBA, R/Statistics is also a really big thing so that was kind of what pulled me in. And it was really fun. A lot of interesting projects and people that I work with were all through that [Twitter] which still surprises me - that I can read a book and tell the author something and they care? It’s weird.
+
+> <h3>I like to make arbitrary rules for myself, one of the things is I don’t tweet stuff that I haven’t read.</h3>
```

Comment:
Please remove \<h3> tags for all pullquotes. For David's just

`> pullquote`

is what is used and it looks good now